cve_bin_tool/version_scanner.py

```diff
@@ -146,6 +146,9 @@ def is_executable(self, filename: str) -> tuple[bool, str | None]:
 
         output: str | None = None
         if inpath("file"):
+            # Ignore special files that are known non-binary
```

Comment:
But `.ntop` files are binary, right? i.e. we merely skip them.

Reply:
Yes should read `non-executable`, will change that